--- /opt/ethos/bin/show	2018-03-03 08:49:15.389180221 -0600
+++ /opt/ethos/bin/show	2018-03-03 08:51:02.061179815 -0600
@@ -12,6 +12,17 @@
 		screen -r "$SCREEN"
 	fi
 }
+
+function show.claymore() {
+
+        if [ "$EUID" -eq "0" ]; then
+                su - ethos -c "screen -wipe"
+                screen -r ethos/
+        else
+                screen -wipe
+                screen -dr "$SCREEN"
+        fi
+}
 case $1 in
 	miner )
 	case "$MINER" in 
@@ -23,6 +34,10 @@
 			show.sgminer
 			SCREEN="sgminer"
 		;;
+		claymore )
+			SCREEN=$(screen -ls | grep '[0-9]*\.claymore' | awk {'print $1'})
+			show.claymore
+		;;
 		* )
 		if [ -z "$2" ]; then
 			echo "tailing /var/run/miner.output (use ctrl+c to exit)"
